src/Eccube/Repository/OrderRepository.php

```diff
@@ -285,15 +285,9 @@ public function getQueryBuilderBySearchDataForAdmin($searchData)
                 ->andWhere('o.id <= :order_id_end')
                 ->setParameter('order_id_end', $searchData['order_id_end']);
         }
-
         // status
         $filterStatus = false;
-        if (!empty($searchData['status']) && $searchData['status']) {
```

Comment:
Please keep this source code.(Don't delete )

Reason
 - There is a possible when plugin use ```get data```